.github/workflows/workflow.yaml

```diff
@@ -12,11 +12,19 @@ on:
 jobs:
   validate:
     name: Validate
-    runs-on: [self-hosted, linux, ARM64]
+    runs-on: ubuntu-latest
     steps:
     - name: Check out code
       uses: actions/checkout@v1
 
+    - name: Add default azure config
+      run: |
+        cat <<EOF > provider.tf
+          provider "azurerm" {
+            features {}
+          }
+        EOF
+
     - name: Run a Terraform init
       uses: docker://hashicorp/terraform:0.12.11
```

Comment:
Use a higher terraform version `0.14.7` preferrably. The version already requires `>= 0.13` 😄